curriculum/challenges/english/01-responsive-web-design/applied-visual-design/decrease-the-opacity-of-an-element.english.md

```diff
@@ -24,7 +24,7 @@ Set the <code>opacity</code> of the anchor tags to 0.7 using <code>links</code>
 ```yml
 tests:
   - text: Your code should set the <code>opacity</code> property to 0.7 on the anchor tags by selecting the class of <code>links</code>.
-    testString: assert(/\.links\s*\{[^}]+opacity\s*:\s*0.7;/.test(code));
+    testString: assert(/\.links\s*\{[^}]+opacity\s*:\s*0?\.7;/.test(code));
```

Comment:
```suggestion
    testString: assert(/\.links\s*\{[^}]*opacity\s*:\s*0?\.7\s*;?\s*}/.test(code));
```

Reply:
@moT01 Are we sure all browsers will end up returning the leading zero in the string?

Reply:
What if we used the `==` operator instead, just in case?

Reply:
I'm not sure all browsers return the same string, it would require some testing. I think I checked on chrome and firefox.

Reply:
@moT01 No, I like what you were doing.  I was just wondering if you could change the suggestion to:
```
    testString: assert(getComputedStyle($('.links')[0]).opacity == '0.7');
```
just in case `getComputedStyle` returns `.7` on some browsers.  The `==` operator would handle that case.

Reply:
done